tests/nl/light_HassTurnOn.yaml

```diff
@@ -15,6 +15,8 @@ tests:
       - "Doe in de keuken de lampen aan"
       - "Mogen in de keuken de lampen aan?"
       - "Zet in de keuken het licht aan"
+      - "Lamp keukenlamp aan"
```

Comment:
This sentence doesn't match with any of the sentences.
Note that spaces are taken into account as of Hassil 1.0

This sentence would match with `<lamp> <area><lamp> aan` or with `<lamp> <name> aan`
Both do not match with any of the intents.